src/tests/user.test.js

```diff
@@ -28,4 +30,21 @@ describe('User related tests:', () => {
     expect(res.status).to.be.equal(401);
     expect(res.body).to.have.property('error');
   });
+  it('should not send a reset link to an invalid email', async () => {
+    const res = await chai.request(app).post('/api/v1/users/forgotPassword').send(mockdata.resetInvalidEmail);
+    expect(res.status).to.be.equal(400);
+  });
+  it('should send a reset link to the user', async () => {
+    const res = await chai.request(app).post('/api/v1/users/forgotPassword').send(mockdata.resetEmail);
```

Comment:
@James-nyagatare make sure that while testing you don't send an email.

Reply:
Implemented @harerakalex .